client/common/cmdline.c

```diff
@@ -3173,6 +3173,11 @@ int freerdp_client_settings_parse_command_line_arguments(rdpSettings* settings,
 			if (!settings->SmartcardLogon)
 				activate_smartcard_logon_rdp(settings);
 		}
+		CommandLineSwitchCase(arg, "x-selection-atom")
```

Comment:
I´m against a new command line option for this, Could be added to `/clipboard:use-selection`

Reply:
I did wonder about that myself, because I couldn't see a way to make this X11-specific CLI option only appear in the X11 version of the code. I'm happy to change the CLI syntax.

Could you clarify your alternative suggestion, please? I'm looking at the command-line code, and it doesn't look to me as if I can add `COMMAND_LINE_VALUE_OPTIONAL` to the existing clipboard option without removing the `COMMAND_LINE_VALUE_BOOL` it already has, which will break existing command lines that use the `+clipboard` or `-clipboard` syntax.

Are you suggesting doing that anyway, or have I misunderstood?